docs/src/_contribute/cli.md

```diff
@@ -0,0 +1,245 @@
+      # The list of tasks must be yaml formatted and consist of a list of strings, list of string lists, or mix of both.
+      meltano job add NAME --tasks '["tap mapper target", "tap2 target2", ...]'
+      meltano job add NAME --tasks '[["tap target dbt:run", "tap2 target2", ...], ...]'
+
+      # Remove a named job
+      meltano job remove NAME
+
+ Read more at https://docs.meltano.com/reference/command-line-interface#jobs
+
+Options:
+  --database-uri TEXT  System database URI.
+  --help               Show this message and exit.
+
+Commands:
+  add     Add a new job with tasks.
+  list    List job(s).
+  remove  Remove a job.
+  set     Update an existing jobs tasks
+```
+
+### Help style guidelines
+
+#### Required items
+
+For required items such as commands and arguments, use text without brackets or braces. In the following examples, all
+words and arguments are required:
+
+```
+meltano run JOB_NAME
+meltano invoke PLUGIN_NAME
+meltano schedule list
+meltano schedule remove SCHEDULE_NAME
+```
+
+#### Optional items
+
+Use square brackets around an optional items. If there's more than one optional item, enclose each item in
+its own set of square brackets. In the following example the `--log-level` and `--dump` are optional, while PLUGIN_NAME
+is required:
+
+```
+meltano [--log-level=LEVEL] invoke [--dump=config] PLUGIN_NAME
+```
+
+#### Mutually exclusive items
+
+Use curly braces (`{}`) to indicate that the user must choose one—and only one—of the items inside the braces. Use
+PIPES (`|`) to separate the items:
+
+```
+meltano schedule {list|remove}
+```
+
+#### Repeating items
+
+Use three trailing dots and no spaces (`...`) to indicate that the user can specify multiple values for the items:
+
+```
+meltano run BLOCKS ...
+meltano install [] [PLUGIN_NAME] ...
+```
+
+### Placeholders and angle brackets
+
+When writing help output its often useful to provide examples that utilize place holders. In complex cases where you're trying illustrate how a user might use feature, its often useful to use actual tap/target/mapping names to illustrate intent. But most commonly, especially for simple examples use common terms e.g. `TAP` rather than `extractor` or `tap-something`. 
+
+We also have significant historic use of angle brackets as place holders for user input in our documentation (and also help out). Which often looks like:
+
+```
+meltano elt <tap> <target>
+```
+
+In the context of something like a more conversational guide or other online docs this does help indicate that we expect the user to supply this input. However, our current use in help output is often ambigous when it comes to indicating whether this is a required item or optional item. So moving forward if you use this form in CLI help out you should still follow the documented conventions for indicating whether something is required:
+
+```
+# TAP and TARGET are upper case since they are required inputs
+meltano run <TAP> <TARGET>
```

Comment:
```suggestion
meltano run <TAP_NAME> <TARGET_NAME>
```

Perhaps we should have `<PLUGIN_NAME>` in there as well? I don't know if using `meltano run` is the best example because technically a tap/target pair isn't required.